Source/WebKit/WebProcess/WebPage/WebFrame.cpp

```diff
@@ -150,18 +150,20 @@ Ref<WebFrame> WebFrame::createSubframe(WebPage& page, WebFrame& parent, const At
     return frame;
 }
 
-Ref<WebFrame> WebFrame::createLocalSubframeHostedInAnotherProcess(WebPage& page, WebFrame& parent, WebCore::FrameIdentifier frameID, WebCore::LayerHostingContextIdentifier layerHostingContextIdentifier)
+Ref<WebFrame> WebFrame::createLocalSubframeHostedInAnotherProcess(WebPage& page, WebFrame& parent, WebCore::FrameIdentifier frameID, WebCore::LayerHostingContextIdentifier layerHostingContextIdentifier, std::optional<ScopeExit<Function<void()>>>&& invalidator)
 {
     auto frame = create(page);
     RELEASE_ASSERT(page.corePage());
     RELEASE_ASSERT(parent.coreAbstractFrame());
-    auto coreFrame = LocalFrame::createSubframeHostedInAnotherProcess(*page.corePage(), makeUniqueRef<WebFrameLoaderClient>(frame.get()), frameID, *parent.coreAbstractFrame());
+    auto coreFrame = LocalFrame::createSubframeHostedInAnotherProcess(*page.corePage(), makeUniqueRef<WebFrameLoaderClient>(frame.get(), WTFMove(invalidator)), frameID, *parent.coreAbstractFrame());
     frame->m_coreFrame = coreFrame.get();
     frame->setLayerHostingContextIdentifier(layerHostingContextIdentifier);
 
     ASSERT(!frame->m_frameID);
     frame->m_frameID = coreFrame->frameID();
     WebProcess::singleton().addWebFrame(frameID, frame.ptr());
+    if (invalidator)
```

Comment:
This is a use-after-move.  I think we can just remove this whole thing.

Reply:
It looks like we might need this because we're creating a new WebFrame as well. I'll fix the use-after-move